app/models/user.rb

```diff
@@ -2,6 +2,8 @@ class User < ApplicationRecord
   has_many :projects, foreign_key: "caseworker"
   has_many :notes
 
+  default_scope { order(first_name: :asc) }
```

Comment:
Okay! `default_scope` is pretty contentious! 

I let the last one slide as I just couldn't come up with a scernario in which Tasks are ordered by anything other that the `order` attribute, but here I can imagine Users being sorted different ways throughout the application and so we should avoid `default_scope`, here is some reading:

https://andycroll.com/ruby/dont-use-default-scope/

Reply:
Interesting 🤔 

Added a separate scope, and called it on the others